scripts/clean-olm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -x
+
+TARGET_NAMESPACE=${TARGET_NAMESPACE:-"default"}
+CSV_VERSION=${CSV_VERSION:-"0.0.1"}
+
+oc delete -n ${TARGET_NAMESPACE} csv osp-director-operator.v${CSV_VERSION}
+oc delete -n ${TARGET_NAMESPACE} subscription osp-director-operator-subscription
+oc delete -n ${TARGET_NAMESPACE} catalogsource osp-director-operator-index
+oc delete -n ${TARGET_NAMESPACE} operatorgroup osp-director-operator-group
```

Comment:
isn't scripts/deploy-with-olm.sh and scripts/clean-olm.sh more a osp-director-dev-tools thing?

Reply:
That's a fair point, as it certainly could be.  I was just following the pattern we used for the `openstack-cluster-operator`, where we had the build/deploy scripts in the operator repo itself: https://github.com/openstack-k8s-operators/openstack-cluster-operator/tree/master/scripts.  Since we're not merging CSVs, however, perhaps it makes sense to just move this script to the tools repo in this project's context.

Reply:
I could go either way. I think that script could be useful as a standalone mechanism to install the eventual (working) operator outside of our dev-tooling environment too. So having it in the primary operator project also makes some sense.